### DISCUSSION TITLE
Updating CSI-Driver

### DISCUSSION THREAD

**Discussion by gr33npr** [kubernetes-csi/csi-driver-smb#632] [Q&A] [answered]
Hello,

I have a running cluster with an initial csi-driver-smb installation (2y and 26d ago). How can i update to the current 1.11 release? I just found manuals for new installations.
Thank you.

Kind regards

**Comment by andyzhangx** [comment, marked as answer]
helm upgrade should work, you could also uninstall and reinstall the driver.
